### ISSUE TITLE
NER Labels

### ISSUE THREAD

**Issue by rensdebee** [wietsedv/bertje#27]
When trying to parse a dutch sentence using transformers NER the only output labels that exist are label_0 and label_1, is this correct and what do these labels mean?

**Comment by wietsedv** [comment]
You are likely using the pre-trained model (`GroNLP/bert-base-dutch-cased`). This model is not fine-tuned for a task like NER. You'll want to fine-tune the model on some labelled data or use a model that is already fine-tuned for NER, like `wietsedv/bert-base-dutch-cased-finetuned-conll2002-ner`. You can see in the [config](https://huggingface.co/wietsedv/bert-base-dutch-cased-finetuned-conll2002-ner/blob/main/config.json) that there are labels.
